tests/Models/Article.php

```diff
@@ -62,4 +63,17 @@ public function getTitleAttribute(string $value): string
     {
         return strtoupper($value);
     }
+
+    /**
+     * Uppercase Content accessor.
+     *
+     * @return Attribute
+     */
+    public function content(): Attribute
+    {
+        return new Attribute(
+            get: fn ($value) => $value,
```

Comment:
This will break with the testsuite. The package in v13 needs to be compatible with php7.4

Reply:
https://github.com/hamedghaderi/laravel-auditing/pull/1

Reply:
Fixed with `method_exists($model, 'hasAttributeMutator')`

Reply:
Thanks to @erikn69, the new patch should fix the issue.